llvm/include/llvm/Analysis/DXILResource.h

```diff
@@ -446,6 +446,10 @@ class DXILBindingMap {
     return Pos == CallMap.end() ? Infos.end() : (Infos.begin() + Pos->second);
   }
 
+  // Resoloves the use of a resource handle into the unique description of that
+  // resource by deduping calls to create.
```

Comment:
Use `///` so doxygen can pick up the comment here. Also, I think this description predates this returning a vector - we should point out that this gives us the set of possible resources in cases that are ambiguous.